### PR TITLE
add Fix Jobs module

### PR DESCRIPTION
**Pull request by ildyria**
Built on top of #2351

### CODE REVIEW

File: app/Livewire/Components/Modules/Maintenance/FixJobs.php [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	public function render(): View
+	{
+		Gate::authorize(SettingsPolicy::CAN_UPDATE, Configs::class);
+
+		return view('livewire.modules.maintenance.fix-jobs');
+	}
+
+	/**
+	 * Return the size used by Lychee.
+	 * We now separate this from the initial get() call as this is quite time consuming.
```
Comment (d7415) [inline app/Livewire/Components/Modules/Maintenance/FixJobs.php]: This description looks wrong?

**Review by d7415 — APPROVED** [review]
Unsure about a comment, otherwise lgtm.

File: app/Livewire/Components/Modules/Maintenance/FixJobs.php (hunk 1 above)
Comment (ildyria) [reply app/Livewire/Components/Modules/Maintenance/FixJobs.php]: indeed.

Comment (ildyria) [reply app/Livewire/Components/Modules/Maintenance/FixJobs.php]: Fixed: [8695d4c](https://github.com/LycheeOrg/Lychee/pull/2352/commits/8695d4cf7e9fb1750ecd308b3fcb2a8a7b0bf62e)

**Pull request merged**